src/main/java/org/jitsi/jicofo/SSRCValidator.java

```diff
@@ -211,6 +211,50 @@ public SSRCValidator(String              endpointId,
         this.logger = Logger.getLogger(classLogger, logLevelDelegate);
     }
 
+    /**
+     * Returns how many sources per media type per participant is allowed in the
+     * conference.
+     * @return an integer
+     */
+    public int getMaxSourceCount()
+    {
+        return this.maxSourceCount;
+    }
+
+    /**
+     * Checks how many sources are current in the conference for given
+     * participant.
+     *
+     * @param owner An owner's JID (can be <tt>null</tt> to check for not owned
+     * sources)
+     * @param mediaType The type of the media for which sources will be counted.
+     * @return how many sources are currently in the conference source map for
+     * given media type and owner's JID.
+     */
+    private long getSourceCountForOwner(Jid owner, String mediaType)
+    {
+        return this.sources.getSourcesForMedia(mediaType)
+            .stream()
+            .filter(
+                source -> Objects.equals(
```

Comment:
is this doing a content comparison? or a reference comparison?

Reply:
It will call .equals which will do content comparison if the first argument is not null. It will also return true if both are null.

Reply:
https://docs.oracle.com/javase/7/docs/api/java/util/Objects.html#equals(java.lang.Object,%20java.lang.Object)